docs/src/index.tsx

```diff
@@ -1,5 +1,9 @@
-import * as React from 'react';
-import * as ReactDOM from 'react-dom';
-import { App } from './App';
+import React from 'react'
+import ReactDOM from 'react-dom/client'
+import {App} from './App.tsx'
 
-ReactDOM.render(<App />, document.getElementById('js-main'));
+ReactDOM.createRoot(document.getElementById('root')!).render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+)
```

Comment:
Drop the namespaces and import the specific functions we need from the modules

```suggeston
import { StrictMode } from 'react'
import { createRoot } from 'react-dom/client'
import {App} from './App.tsx'

createRoot(document.getElementById('root')!).render(
  <StrictMode>
    <App />
  </StrictMode>,
)
```

Reply:
thank you! It's just like what you said. Use named imports!

Reply:
https://github.com/remeda/remeda/pull/359/commits/92b0107dca6585598b21af04b84d1f0734788498

used named import it!